source/includes/Overview/_changelog.md.erb

```diff
@@ -17,6 +17,7 @@
 | 2019-08-12 | Updating support links |
 | 2019-09-04 | Adding the 'name' filter to the job codes GET operation |
 | 2019-09-23 | Updating code templates to allow overriding base URL and headers. |
+| 2020-06-17 | Updating request formats to make response code documentation more accurate. |
```

Comment:
Can you put this at the bottom of the list? Actually I'll just do it right now while I have your code checked out.